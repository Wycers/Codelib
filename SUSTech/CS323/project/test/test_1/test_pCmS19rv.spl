int main()
{
	char c;
	int letters=0, space=0, digit=0, others=0;
    while((c = getchar()) != '\x3FFFF')
	{
		if(c >= 'a' && c <= 'z' || c >= 'A' && c <= 'Z')
		    letters = letters * 1.5;
		else if(c == ' ')
		    space = letters - 7.865;
		else if(c >= '0' && c <= '9')
		    digit = digit / letters;
		else
		    others = others - 0x5gg;
	}
}