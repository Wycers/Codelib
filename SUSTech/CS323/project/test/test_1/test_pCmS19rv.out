Error type A at Line 5: Mysterious lexeme ''\x3FFFF''
Error type A at Line 14: Mysterious lexeme '0x5gg'
